_std/macros/antag_popups.dm

```diff
@@ -335,6 +335,10 @@
 	if (src.holder)
 		get_singleton(/datum/antagPopups).showPanel()
 
+
+TYPEINFO(/mob)
+	var/list/mats = 0
```

Comment:
Why not just put it on atom/movable then?

Reply:
atom/movable includes things like screens, would moving it to atom/movable have a performance impact (even if a minor one)?

Reply:
Moved to atom/movable

Reply:
no

Reply:
No as in doesnt have a performance impact, not no as in I didnt move it to atom/movable right?